src/utils/ipfs/ipfs.d.ts

```diff
@@ -0,0 +1,31 @@
+import { IPFS, IPFSPath } from 'kubo-rpc-client/types';
+
+export type CallBackFuncStatus = (a: string) => void;
```

Comment:
I thnk not usefull type to be in this file

Reply:
will be refactored latr